Library/Formula/libcppa.rb

```diff
@@ -1,42 +1,82 @@
-require "formula"
-
 class Libcppa < Formula
+  # TODO: since libcppa has been renamed to CAF, this formula should eventually
+  # be renamed to 'caf.rb'.
   homepage "http://actor-framework.org/"
-  url "https://github.com/actor-framework/actor-framework/archive/0.11.0.tar.gz"
-  sha1 "202f2fd72a5af59d7ace6b7300df1fcc19f1857f"
+  url "https://github.com/actor-framework/actor-framework/archive/0.12.0.tar.gz"
+  sha256 "de927f53f5765489406d636dcf042bf3f281bc75ebfe4fce933dc1d9cff60481"
+  head "https://github.com/actor-framework/actor-framework.git"
 
-  # since upstream has rename the project to actor-framework (or libcaf in its
-  # pkgconfig file), we need to rename libcppa to libcaf in the future
+  # Since CAF does not produce development releases, we repurpose --devel to
+  # install from the 'develop' branch.
+  devel do
+    url "https://github.com/actor-framework/actor-framework.git",
+      :branch => "develop"
+    version "0.13-devel"
+  end
```

Comment:
This isn't how we use `devel`; just remove this and you could change `head` to point to that branch if you'd rather.